### ISSUE TITLE
Inaccurate explanation of closing issue with PR in process.md

### ISSUE THREAD

**Issue by wkurniawan07** [TEAMMATES/teammates#5603]
This is from a section in `process.md`:

```
In the PR description, mention the issue number in this format: Fixes #1760.
Doing so will create an automatic reference from the issue to the pull request.
```

This is _inaccurate_: automatic reference is achieved by simply mentioning the issue number with the `#`. What is intended here is [closing the issue when the PR is merged](https://github.com/blog/1506-closing-issues-via-pull-requests).

**Comment by AakanxuShah** [comment]
Hi @wkurniawan07 , I am new to open source contribution and I would like to work on this issue !
I have already read TEAMMATES Contribution Orientation Guide. 

For this issue, I need to create a fork of this project  to commit proposed changes to. Submitting a change to this file will write it to a new branch in my fork, so I can send a pull request - is that right ?

**Comment by wkurniawan07** [comment]
Hi @AakanxuShah, that's right, remember to read through our [development process document](https://github.com/TEAMMATES/teammates/blob/master/devdocs/process.md) to get what is expected from the pull request.
Also, I have noticed that you commented on several other issues; unfortunately, they are all marked `d.FirstTimers`, and one contributor is limited to one such issue so you have to make a choice on which one to work on.

**Comment by AakanxuShah** [comment]
@wkurniawan07 : Thank you for you prompt reply ! Okay then I will work on this one. 

In this issue , I need to change the explanation from " Doing so will create an automatic reference from the issue to the pull request." to "To close an issue from a Pull Request, all you have to do is include the special keyword syntax (eg. "fixes #5") in the body of your Pull Request.And the referenced issue will automatically be closed when the PR is merged into the default branch." - is this correct ?
